packages/perseus/src/components/lint.tsx

```diff
@@ -1,10 +1,11 @@
 import {StyleSheet, css} from "aphrodite";
 import * as React from "react";
-import ReactDOM from "react-dom";
+import Tooltip from "@khanacademy/wonder-blocks-tooltip";
```

Comment:
I needed to migrate to the WB tooltip as the old one was getting clipped by the preview element(s).  Moving to WB also simplifies this component quite a bit.